src/openrct2-ui/windows/Options.cpp

```diff
@@ -1007,26 +1028,9 @@ class OptionsWindow final : public Window
 
     void DisplayDraw(rct_drawpixelinfo* dpi)
     {
-        DrawTextBasic(
-            dpi, windowPos + ScreenCoordsXY{ 10, widgets[WIDX_FULLSCREEN].top + 1 }, STR_FULLSCREEN_MODE, {}, { colours[1] });
-
-        // Disable resolution dropdown on "Windowed" and "Fullscreen (desktop)"
```

Comment:
Does it still properly grey out the mentioned options after removing this code?

Reply:
Yes, that's done by adding/removing it from `disabled_widgets` in `DisplayPreDraw()`. (And I just double-checked it works to make sure.)